### ISSUE TITLE
Problems with booting up GodMode9 cia/3dsx 

### ISSUE THREAD

**Issue by ghost** [SUOlivia/A9SP#9]
I am using a New 3DS on 11.6 Luma B9S. I have followed all of the directions exactly with the latest releases and this still isn't working correctly. This is in reference to both the GodMode9 cia and 3dsx. The cia just turns black and goes to the home menu while the 3dsx says that it can't find boot0.firm even when it is clearly on the sd card.

**Comment by SUOlivia** [comment]
Is the Luma firm renamed as boot0.firm? Also, assuming that it doesn't work on N3ds is misleading, I do 100% of my testing on N3ds and even need to switch firm0s quite often to be sure it works with b9s.

**Comment by ghost** [comment]
Yes, it is the Luma firm that is renamed. Also, sorry for misleading title. Will change. Also, I just want to thank you for your hard work.

**Comment by SUOlivia** [comment]
Are you using FastBoot3DS? If so, can you give me the slot that a9sp uses? And I'll send some more updated builds later

**Comment by ghost** [comment]
No, right now I am using B9S. I was waiting for this update to switch. Sorry

**Comment by SUOlivia** [comment]
@Leafgreen26 can you try these builds?
chainloader : https://puu.sh/z36U4/2ac83eafb5.firm
userland : https://puu.sh/z36Wc/390d2110a6.cia

**Comment by ricksee** [comment]
@Leafgreen26 I had the same problem, it just needed to be updated. Download the new updates from above. Also change your original boot.firm to boot0.firm. Then, change A9SP.firm to boot.firm. Finally, install the CIA.

**Comment by ghost** [comment]
Thanks. The new builds work.

**Comment by Eclipse-Sol** [comment]
was having the same issue, can confirm it works

**Comment by ghost** [comment]
So are we going to get a new release soon, or are all of these problems already updated with the last release? Sorry not really familiar with Github.

**Comment by ghost** [comment]
Also, while the new cia version works, I can't get the 3dsx version to work.

**Comment by ghost** [comment]
Last thing that I forgot to mention. The build version provided above is super slow compared to older versions where it was almost instant. It takes like 10-15 seconds to load now which is not a lot but compared to previous versions it is.
